srcpkgs/xwinwrap/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'xwinwrap'
+pkgname=xwinwrap
+version=0.9
+revision=1
+build_style=gnu-makefile
+makedepends="libXrender-devel"
+short_desc="X11 Window as background"
+maintainer="SolitudeSF <solitudesf@protonmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/Aaahh/xwinwrap"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=0e840586deaf6780d426dc49f7a344366198aead96535cc1eb2ac7ce366777df
+
+pre_install() {
+	sed -i 's/\/local//' Makefile
```

Comment:
please use vsed, see Manual.md

Reply:
ah, had this template lying around since forever, havent refreshed it properly